dts/bindings/mtd/jedec,spi-nor.yaml

```diff
@@ -24,3 +29,24 @@ properties:
     type: phandle-array
     required: false
     description: RESETn pin
+  page-size:
+    type: int
+    required: false
+    default: 256
+    description: |
+      Page size in bytes. This property is not respected if a supported exact
+      device name is given as a compatible.
+  erase-types:
```

Comment:
At least one candidate chip that needs this sort of thing (micron,mt25qu01gbbb) has a non-standard chip-erase command (0xC4 instead of 0xC7).  So we should have a chip-erase-cmd property as well that gets stored into `spi_nor_config`.

Reply:
On further inspection: This chip has two stacked dies. 0XC4 can only erase on of them at a time and needs an address with the command. This means the current way we use chip erase does not work with this device. One way would be to treat the die erase command as an erase type in the erase types array (in this case with erase size = size/2) and have a flag which disables chip erase. Are you aware of any other device that has non standard chip erase commands? If not, i propose to drop the chip_erase_cmd property and add a has_chip_erase property.

Reply:
I guess we could do that for now, and revisit it if it turns out that there's a use case for chip-erase that can't be handled by iterating through block erases.